### ISSUE TITLE
How can I get date(6/28/1968), if in property 'jd' kept  count days

### ISSUE THREAD

**Issue by ArtemShapovalov** [deadprogram/hebruby#1]
Example: Hebruby::HebrewDate.new(10,2,5776).jd = 2457527

**Comment by deadprogram** [comment]
Hello @artemshapovalov 

This gem needed some serious maintenance to update it for newer Ruby versions, so I've made some corrections and released an update (2.1.0).

Now you can do this:

``` ruby
require 'hebruby'

hebrew_date = Hebruby::HebrewDate.new(2, 4, 5728)
puts hebrew_date.julian_date  # => 1968-06-28
```

I've added a couple examples under `examples`.

Hope that helps!

**Comment by ArtemShapovalov** [comment]
Thanks for the help, this is what I needed!
